### PR TITLE
Update centralized-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
added a note that MFA is not supported when using the Central Deployment for Addins Powershell module.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3077

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -129,7 +129,8 @@ When the tool finishes running, it produces an output file in comma-separated (.
     
 - Supported Mailbox - If they are on an OAuth-enabled mailbox
 
-
+> [!NOTE]
+> MFA is not supported when using the Central Deployment for Addins Powershell module.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
> Multifactor authentication is not supported when using the Central Deployment for add-ins PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Pull request merged**